lib/net/http/header.rb

```diff
@@ -33,14 +176,30 @@ def size   #:nodoc: obsolete
 
   alias length size   #:nodoc: obsolete
 
-  # Returns the header field corresponding to the case-insensitive key.
-  # For example, a key of "Content-Type" might return "text/html"
+  # Returns the string field value for the case-insensitive field +key+,
+  # or +nil+ if there is no such key;
+  # see {Fields}[rdoc-ref:Net::HTTPHeader@Fields]:
+  #
+  #   req = Net::HTTP::Get.new(uri)
```

Comment:
I feel like if you want to highlight the case-insensitiveness of this method, it would be useful to have a few examples of that here rather than in the documentation for `Net::HTTP::Header`.

Reply:
I'm thinking to change all the keys to capitalized strings, with no lowercased and no symbols.  That's what people are used to seeing.  I don't think many care about the case-insensivity (though the description should still mention it).